README.rst

```diff
@@ -49,18 +49,20 @@ Install or update dependencies::
     $ brew install libevent libmagic libxml2 libxslt openssl postgresql graphviz nginx python3
     $ brew install freetype libjpeg libtiff littlecms webp  # Required by Pillow
     $ brew cask install adoptopenjdk8
-    $ brew install elasticsearch@5.6 node@10
+    $ brew install elasticsearch@6.8 node@10
 
 NOTES:
 
+* To migrate from ES5 to ES6::
+
+    $ brew unlink elasticsearch@5.6
+    $ brew install elasticsearch@6.8
+    $ brew link elasticsearch@6.8
```

Comment:
`brew link elasticsearch@6.8` works but `brew link --force elasticsearch@6.8` is better IMO if user runs install-uninstall-link trio over and over again.